app/i18n/locales/pt-BR.json

```diff
@@ -740,7 +740,7 @@
   "sending_email_confirmation": "enviando email de confirmação",
   "Sending_to": "Envio para",
   "Server": "Workspace",
-  "Server_version": "Versão da workspace: {{version}}",
+  "Server_version": "Espaço de Trabalho: {{version}}",
```

Comment:
Besides the regression of not using sentence case (it should be "Espaço de trabalho" with a lower t), you're confusing users here.
On "server" string, you're calling it "workspace".
On "server_version", you're calling it "Espaço de trabalho".

We should use the same terms across the whole app.
It's not an easy task to do, because we don't have control over other languages, like finish, but we should be careful with en, es and pt-BR.
Talk to design team to understand if there's any official term for workspace in pt-BR, but I don't know why you changed these strings on this PR, since it doesn't look related to ellipised text.